room.js

```diff
@@ -1,703 +0,0 @@
-            console.log('\n \x1b[33m%s\x1b[0m', 'You\'ve already removed the boards!')            
-        }
-    },
-    unlock(){
-        if(this.locked === true && player.inv.includes('drawer key')){
-            this.locked = false
-            this.desc = ('\n It\'s a box boarded up now without that pesky metal lock in the way!')
-            room1.south_description = "The box has forgone its relationship with his pal the metal lock."
-            console.log('\n \x1b[33m%s\x1b[0m', 'The metal lock falls to the ground upon being unlocked with a loud THUD')
-        } else if(this.locked === false){
-            console.log("\n It's already unlocked!")
-        } else {
-            console.log("You need a key!")
-        }
-    },
-    open(){
-        if(this.boarded === false && this.locked === false){
-            room1.south_description = "The box only has a note in it now, with broken planks and a metal lock on the ground!"
-            this.desc = "\nThe chest you just opened to get the note!"
-            console.log('\n \x1b[33m%s\x1b[0m', 'You found a hidden note with that says... \"The last numbers are 730!\"')
-        } else {
-            console.log("You can't open this yet!")
-        }
-    }
-}
-
-
-let room1_cardboard_box = {                                 
-    opened: false,
-    current_room: 'room1',
-    current_facing: 'west',
-    visible: true,                                    
-    // takeable: false,
-    useable: true,                  
-    desc: "\nIt's a cardboard box that appears to be slightly open... On the side there's a strange, three-pronged symbol.",
-    open(){
-        if(this.opened === false){
-            this.opened = 'true'
-            this.desc = "The box is now open and empty"
-            console.log("\nYou found a crowbar! Current inventory: ")
-            player.inv.push('crowbar')
-            console.log(player.inv)
-            room1.west_description = "\nThere is an opened cardboard box in the west side now..."
-            
-        } else {
-            console.log("\nIt's already open!")        // To make the output look cleaner I want to put new line breaks 
-        }
-        
-    },
-    take(){
-        console.log("\nYou can't take this! ")
-    },
-
-}
-
-
-let hay = {
-    name: "Hayloft",
-    desc: "I am sorry but the hayloft is out of reach!",
-    current_room: 'room3',
-    current_facing: 'east',
-    inv: ["horseshoe"],
-    reachHay() {
-        if (player.inv.includes('ladder')) {
-            return console.log("You have reached the hayloft and now you see a horseshoe. It must be another clue!")
-        } else {
-            return console.log("You need to reach the hayloft first!")
-        }
-    },
-    takeHorseshoe() {
-        let horseshoe = this.inv.pop()
-        player.inv.push(horseshoe)
-        return console.log("The numbers 12345 are inscribed on the horseshoe")
-    }
-}
```

Comment:
This is another section that is unused. If there are plans to use it, adding a comment saying what it will be doing could be a good idea. Otherwise removing it would help minimize your lines of code